README.md

```diff
@@ -12,6 +14,113 @@ Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 {:membrane_element_udp, "~> 0.2"}
 ```
 
+## Usage example
+
+The example below shows 2 pipelines - `UDPDemo.Send` downloads an example file over HTTP and
```

Comment:
I would personally swap `-` for `.` and treat this sentence parts as two sentences. At least it makes more sence to me in that form.